distribution/docs/src/main/resources/content/_running/command-scheduler.adoc

```diff
@@ -54,7 +52,7 @@ Scheduled commands have the following syntax: `${ddf-branding-lowercase}.platfor
 . Find the desired configuration to modify, and update fields.
 . Click the *Save changes* button.
 
-====== Command Output
+=== Command Output
```

Comment:
This section should no be under `Command Scheduler`. Is would fit better under `Console Commands`.

Reply:
I'm not sure. It seems to be about the difference in output between manually-run commands and scheduled ones.

Reply:
Oh, you're right:+1:

Reply:
That sounds like the right answer